### PR TITLE
Examples for setting up custom service executors

### PR DESCRIPTION
**Pull request by Aklakan**
Basic examples for custom service executors; adding them here with the intent of linking to them from jena-site.

### CODE REVIEW

File: jena-examples/src/main/java/arq/examples/service/CustomServiceExecutor.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import org.apache.jena.sparql.exec.QueryExec;
+import org.apache.jena.sparql.exec.QueryExecDatasetBuilder;
+import org.apache.jena.sparql.service.ServiceExecutorFactory;
+import org.apache.jena.sparql.service.ServiceExecutorRegistry;
+import org.apache.jena.sparql.util.Context;
+import org.apache.jena.sparql.util.QueryExecUtils;
+
+public class CustomServiceExecutor {
+
+	/** Query for resources having the label "Apache Jena"en */
```
Comment (afs) [inline jena-examples/src/main/java/arq/examples/service/CustomServiceExecutor.java:39]: Tab!

Comment (afs) [reply jena-examples/src/main/java/arq/examples/service/CustomServiceExecutor.java:39]: "Apache Jena"en => "Apache Jena"@en

**Review by afs — APPROVED** [review]
Could you squash it down to one commit "Examples for setting up custom service executors" please? The other two are short term development items.

**Review by rvesse — APPROVED** [review]

**Comment by Aklakan** [comment]
Now squashed

**Review by kinow — APPROVED** [review]

**Pull request merged**